pkg/database/authorized_app.go

```diff
@@ -264,6 +264,18 @@ func (db *Database) SaveAuthorizedApp(a *AuthorizedApp, actor Auditable) error {
 				audits = append(audits, audit)
 			}
 
+			if existing.APIKey != a.APIKey {
```

Comment:
The API key would never change, its immutable and also one-way hashed. I also don't want the API key to appear in a logl.

Reply:
/close